Fix Idagio connector

LibertyFrog opened the pull request:
**Describe the changes you made**
Fixed the Idagio connector not recognizing when a track is playing.

**Additional context**
The Idagio connector was not scrobbling music. The `isPlaying` check expects the play button label to equal "Pause" when a track is playing which is also what is visible in the DOM, however they use a `text-transform: uppercase` CSS styling on it.
I call `toUpper` on the found text, so it doesn't break again should they remove the styling.

hornbuckle commented:
If the text inside the element is "Pause," CSS text-transform shouldn't have any effect on the connector script since it would only affect how the browser renders the text.
Can you provide a link to show the issue you're trying to fix?

LibertyFrog commented:
> If the text inside the element is "Pause," CSS text-transform shouldn't have any effect on the connector script since it would only affect how the browser renders the text. Can you provide a link to show the issue you're trying to fix?

That is not the behaviour I'm observing. As for a link: go to https://app.idagio.com and click the play icon on any album tile. When I do it the extension doesn't start scrobbling and when I look at the `State` object I see the `isPlaying` property is still `false`.

This is what I see in the DOM for the pause button selector, "Pause":
![pause-in-dom](https://github.com/web-scrobbler/web-scrobbler/assets/124195606/f40ca688-6d8a-4b66-8866-4d623ca68829)

This is the CSS applying the `text-transform`:
![css-class](https://github.com/web-scrobbler/web-scrobbler/assets/124195606/2f1965b2-ed5e-4065-9392-e18aa0d03e0b)

I added logging to the isPlaying getter (without the `toUpper` I added later):
![log-text](https://github.com/web-scrobbler/web-scrobbler/assets/124195606/2a01c068-311e-4803-90d7-49f13bfd3310)

This is the log output, all uppercase "PAUSE":
![log-value](https://github.com/web-scrobbler/web-scrobbler/assets/124195606/e28ac5ae-4220-40e9-96ce-2f5887d76205)

This is the log output after toggling `text-transform` in the browser dev tools (no other change):
![css-no-text-transform](https://github.com/web-scrobbler/web-scrobbler/assets/124195606/ec25457e-1909-4cb9-9a6f-767febcfb458)
![log-value-no-text-transform](https://github.com/web-scrobbler/web-scrobbler/assets/124195606/bd966717-6878-482b-a569-56fe20678995)

inverse commented:
Any updates on this one?

yayuyokitano commented:
Merging this. Improvement suggested by hornbuckle can be applied in a separate PR.
